src/js/plugins/menu_toggle/plugin.menu_toggle.js

```diff
@@ -94,26 +103,24 @@
         'BookReader:resize'
       ];
 
-      $(document).on(eventsToHandle.join(' '), fullscreenEventRegister);
-      $(window).on('orientationchange', fullscreenEventRegister);
+      $(document).on(eventsToHandle.join(' '), menuToggleEventRegister);
+      $(window).on('orientationchange', menuToggleEventRegister);
+      $(document).on('BookReader:fullscreenToggled', setupDOMandHandlers);
+      $(window).on('DOMContentLoaded', setupDOMandHandlers);
```

Comment:
suggestion: Replace the `setupDOMandHandlers` with a direct call to `setupNavForToggle`:

```js
$(window).on('DOMContentLoaded', () => setupNavForToggle(br));
```